tests/cmd/e2e/main.go

```diff
@@ -142,6 +142,7 @@ func main() {
 		cluster2.UpgradeAll(upgradeVersions[0])
 		oa.UpgradeTidbClusterOrDie(cluster2)
 		oa.CheckUpgradeOrDie(ctx, cluster2)
+		oa.CheckManualPauseTiKVOrDie(cluster2)
```

Comment:
For TiKV, the `CheckUpgradeOrDie ` method checked the upgrade progress, so you should move `CheckManualPauseTiKVOrDie` to `CheckUpgradeOrDie ` method.